.github/workflows/mobile-traffic_director.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: 'Run GcpTrafficDirectorIntegrationTest'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GCP_JWT_PRIVATE_KEY: ${{ secrets.GCP_SERVICE_ACCOUNT_JWT_TOKEN }}
+        GCP_TEST_PROJECT_API_KEY: ${{ secrets.GCP_TEST_PROJECT_API_KEY }}
```

Comment:
im wondering if we want to change the name here, or just the secret tht is used - seems like a temp name

Reply:
discussed offline, the reason I put "test project" is to make clear that the CI job is connecting to a test GCP project that we created internally in google, i.e. it is fake data and will never contain user/customer data